virttest/env_process.py

```diff
@@ -630,6 +631,13 @@ def preprocess(test, params, env):
     if params.get("storage_type") == "nfs":
         image_nfs = nfs.Nfs(params)
         image_nfs.setup()
+        distro_details = distro.detect()
+        if distro_details.name.upper() != 'UBUNTU':
+            params['set_sebool_local'] = "yes"
+            params['local_boolean_varible'] = "virt_use_nfs"
+            params['local_boolean_value'] = "on"
+            seLinuxBool = utils_misc.SELinuxBoolean(params)
+            seLinuxBool.setup()
```

Comment:
I just notice that we may have some negative test case, which disable 'virt_use_nfs' intentionally. So it's better to get these params first, and set a default value:
```
set_sebool_local =  params.get('set_sebool_local', 'yes')
...
```